axonserver/src/main/java/io/axoniq/axonserver/transport/grpc/EventProcessorGrpcController.java

```diff
@@ -80,11 +85,18 @@ public EventProcessorGrpcController(EventProcessorAdminService service,
     @Override
     public void pauseEventProcessor(EventProcessorIdentifier processorId,
                                     StreamObserver<AdminActionResult> responseObserver) {
+        if (authenticationProvider.get().isAuthenticated() && !axonServerAccessController.allowed(
+                EventProcessorAdminServiceGrpc.getPauseEventProcessorMethod().getFullMethodName(),
+                processorId.getContextName(),
```

Comment:
processorId.getContextName is not required, other clients create a connection for a context and send the request to that client. If processorId.getContextName is empty use the context from the context provider.